marklogic-data-hub/src/main/resources/ml-config/databases/final-database.json

```diff
@@ -1,5 +1,9 @@
 {
   "database-name": "%%mlFinalDbName%%",
+  "path-namespace" : [ {
```

Comment:
I'm now remembering that if we modify this file in particular, we have to update HubProjectImpl:upgradeProject so that this addition is made programmatically. That's because we can't just overwrite this file, as the user may have made changes in it. 

For that reason, I'm thinking it's better to add the es path-namespace to the XML fields files and then enhance DeployDatabaseFieldsCommand to ensure that the es path-namespace is added if it doesn't exist yet. And we'll need to add tests to verify that it properly adds it or doesn't add it.